lms/services/lti_h.py

```diff
@@ -24,12 +25,18 @@ class LTIHService:
     def __init__(self, _context, request):
         self._context = request.context
         self._request = request
+        self._authority = self._request.registry.settings["h_authority"]
         self._lti_user = request.lti_user
```

Comment:
If the `LTIUser` already contained the authority we would not need to add this here.